docs/References/Runme_Parameters.md

```diff
@@ -0,0 +1,154 @@
+## `freemocap.RunMe()` Specify recording session  paramters 
+
+The `freemocap.RunMe()` function takes a number of parameters that can be used to alter it's default behavior in important ways. Here are the default parameters along with a followed by a brief description of each one. 
+
+---
+### RunMe - Default parameters
+```python
+#in `freemocap/fmc_runme.py`
+def RunMe(sessionID=None,
```

Comment:
need to add the `used_saved_calibration`, `place_skeleton_on_origin` and `save_annotated_videos` flags to this list